docker/supabase/docker-compose.yml

```diff
@@ -0,0 +1,158 @@
+# Usage
+#   Start:          docker-compose up
+#   With helpers:   docker-compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml up
+#   Stop:           docker-compose down
+#   Destroy:        docker-compose -f docker-compose.yml -f ./dev/docker-compose.dev.yml down -v --remove-orphans
+
+version: '3.8'
+
+services:
+  studio:
+    container_name: supabase-studio
+    image: supabase/studio:latest
+    restart: unless-stopped
+    ports:
+      - ${STUDIO_PORT}:3000/tcp
```

Comment:
So we're opening port 3000.  We'll have to evaluate this in staging and production, cc @manekenpix

Reply:
@humphd, I wonder how they still work running on the same port?

I am think if we could use Kong to replace Nginx?

Reply:
We do our SSL through nginx, so I'm not really wanting to replace it.  We just need to have this work behind nginx similar to how Traefik does now.

Reply:
Can I keep this change?

Reply:
For now, sure.  When we put it staging, we'll have to rework this.